### DISCUSSION TITLE
I am getting validation_not_unique error without defining any unique constrains.

### DISCUSSION THREAD

**Discussion by devloper16** [pocketbase/pocketbase#4423] [Q&A] [answered]
I am using version  v0.21.3 of pocketbase. I have a base collection and in it I have a column named url_id (it is a string of "user_id+hash of the url"). The only enable (for both columns url and url_id) is Nonempty that it is a non empty, what is causing this behaviour.

column url is of type URL and url_id is of type plain text.

This is the full error message I am getting (I am using thunder client for Vscode)

{
  "code": 400,
  "message": "Failed to create record.",
  "data": {
    "url": {
      "code": "validation_not_unique",
      "message": "Value must be unique."
    },
    "url_id": {
      "code": "validation_not_unique",
      "message": "Value must be unique."
    }
  }
}

My questions is,

what is causing this behaviour?

Thanks for help,
Dev.

**Comment by ganigeorgiev** [comment, marked as answer]
> I am getting validation_not_unique error without defining any unique constrains

I doubt it. Double check the defined collection indexes under the "Unique constraints and indexes" section. If you have created the unique index manually (eg. via a raw query) and not through the collection model then it is possible to not show in the Admin UI as we currently doesn't load the indexes dynamically.

**Reply by devloper16** [reply]
Sorry my bad I had a unique index set on the column "url_id" made it turned it off and problem is resolved. sorry for such a oversight
